google/cloud/internal/call_context.h

```diff
@@ -70,12 +77,10 @@ class ScopedCallContext {
   static void* operator new[](std::size_t) = delete;
 
  private:
+  OptionsSpan options_span_;
 #ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
-  using ScopedSpan = opentelemetry::trace::Scope;
-  ScopedSpan span_;
+  opentelemetry::trace::Scope otel_scope_;
 #endif  // GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
-  using ScopedOptions = OptionsSpan;
-  ScopedOptions options_;
```

Comment:
I prefer the old names. I think the `using` directives helped disambiguate which "span" we are talking about.

Reply:
The old names and aliases confused at least one person (me).

I wanted to remove the default constructor, but it is used in too many places to do in one pass and the new constructor unblocks cleaning up AsyncRetryLoop and AsyncRestRetryLoop.